jr-objects/src/main/java/com/fasterxml/jackson/jr/ob/impl/BeanPropertyIntrospector.java

```diff
@@ -145,6 +149,15 @@ private static void _introspect(Class<?> currType, Map<String, PropBuilder> prop
                         name = decap(name.substring(2));
                         _propFrom(props, name).withIsGetter(m);
                     }
+                } else if(USE_FIELD_NAME_GETTERS.enabledByDefault()){
```

Comment:
As with other comment, `enabledByDefault()` is not to be used here -- we need actual configuration settings.